doc/api/fs.md

```diff
@@ -2019,11 +2019,17 @@ The file is created if it does not exist.
 
 * `'ax'` - Like `'a'` but fails if `path` exists.
 
+* `'as'` - Open file for appending in synchronous mode.
```

Comment:
Regarding the phrasing here … I’m not sure, but I think this might lead people to believe that `stream.write('foo');` writes the data synchronously, when in reality it still passes it off to the libuv threadpool.

How about something like ``Open a file for appending with the `O_SYNC` flag set, see open(2) for more details.``?

Reply:
I think the meaning of `synchronous mode` is addressed in the `'rs+'` mode above. Would it suffice If i can reference to that? How do i do that in markdown?

Reply:
@SirR4T You can add an anchor around `rs+` with `<a id="rs-flag"></a>` and reference that as ``[`rs+`](#rs-flag)``